src/Dofs/ConstraintHandler.jl

```diff
@@ -76,6 +76,7 @@ struct ConstraintHandler{DH<:AbstractDofHandler,T}
     prescribed_dofs::Vector{Int}
     free_dofs::Vector{Int}
     inhomogeneities::Vector{T}
+    affine_inhomogeneities::Dict{Int,T}
```

Comment:
Is it better to make it  a vector instead (same size as `inhomogeneities`), since we will only acces it when looping over the precribed_dofs?